### PR TITLE
Update 06-running-tests.md

### PR DESCRIPTION
**Pull request by feversocial**
#### Change log

- Running tests documentation.

### Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/02-development/06-running-tests.md [hunk 1]
```diff
@@ -4,47 +4,32 @@ This document describes how to run tests in different ways.
 
 ## What is a testing mode?
 
-Docker SDK allows you to run an application in an environment configured for testing.
+Docker SDK allows you to switch the application into the special mdoe, optimised for running your tests.
```
Comment (alexanderM91) [inline docs/02-development/06-running-tests.md:7]: @andriitserkovnyi could you review it and correct mistypes.

Comment (alexanderM91) [reply docs/02-development/06-running-tests.md:7]: I am not sure that it's special mode, as for me it's configured and prepared environment for running tests.

File: docs/02-development/06-running-tests.md [hunk 2]
```diff
@@ -4,47 +4,32 @@ This document describes how to run tests in different ways.
...
 
-In the testing mode, docker/sdk set of containers configured for testing. For example:
-1. background jobs are disabled;
-2. the webdriver container is present.
+The aim of the testing mode - is to allow you running tests in isolation, having a full control on the system under test and have all needed tooling in place. Once activating testing mode, the following happens:
+1. CLI, applicatoin and webserver contaienrs are switched to the "testing mode";
+2. Background jobs are disabled;
```
Comment (alexanderM91) [inline docs/02-development/06-running-tests.md:11]: To provide data consistency and full isolation.

File: docs/02-development/06-running-tests.md [hunk 3]
```diff
@@ -4,47 +4,32 @@ This document describes how to run tests in different ways.
...
+3. The webdriver container for the end-to-end tests is created (if not yet) and launched.
```
Comment (alexanderM91) [inline docs/02-development/06-running-tests.md:12]: The webriver is up and running.

**Comment by alexanderM91** [comment]
Hey @feversocial, 
thanks a lot for you contribution.
With your updates the document looks :+1:

File: docs/02-development/06-running-tests.md (hunk 1 above)
Comment (feversocial) [reply docs/02-development/06-running-tests.md:7]: I agree! We can leave that as-is. My main concern was rather in the "runs a separate container for testing".

File: docs/02-development/06-running-tests.md (hunk 2 above)
Comment (feversocial) [reply docs/02-development/06-running-tests.md]: fixed

File: docs/02-development/06-running-tests.md (hunk 3 above)
Comment (feversocial) [reply docs/02-development/06-running-tests.md]: fixed

File: docs/02-development/06-running-tests.md (hunk 1 above)
Comment (feversocial) [reply docs/02-development/06-running-tests.md]: fixed

**Review by alexanderM91 — APPROVED** [review]

**Comment by alexanderM91** [comment]
Hi, @andriitserkovnyi, could you please take a look at the PR when you will have some spare time, thank you.

File: docs/02-development/06-running-tests.md [hunk 4]
```diff
@@ -4,47 +4,32 @@ This document describes how to run tests in different ways.
...
 ## What is a testing mode?
 
-Docker SDK allows you to run an application in an environment configured for testing.
+Docker SDK allows you to run your application in an environment, configured for running your tests.
 
-In the testing mode, docker/sdk set of containers configured for testing. For example:
-1. background jobs are disabled;
-2. the webdriver container is present.
+The aim of the testing mode - is to allow you running tests in isolation, having a full control on the system under test and have all needed tooling in place. Once activating testing mode, the following happens:
+1. Application is ready for the "testing mode";
```
Comment (andriitserkovnyi) [inline docs/02-development/06-running-tests.md:10]: How is it "ready"? Is it prepared for the testing mode? If yes, then how? Is it switched into the testing mode?

File: docs/02-development/06-running-tests.md [hunk 5]
```diff
@@ -4,57 +4,52 @@ This document describes how to run tests in different ways.
...
 ## What is a testing mode?
 
-Docker SDK allows you to run an application in an environment configured for testing.
+The Docker SDK lets you run applications in an environment configured for running tests.
 
-In the testing mode, docker/sdk set of containers configured for testing. For example:
-1. background jobs are disabled;
-2. the webdriver container is present.
+In the testing mode, you can run tests in isolation, with full control of the system tested and all needed tooling in place. Once you activate the testing mode, the following happens:
+1. The scheduler is enabled.
```
Comment (alexanderM91) [inline docs/02-development/06-running-tests.md:10]: Please combine 1 and 3 into
The scheduler is enabled (Background jobs should be stopped for preserving data consistency and full isolation)

**Pull request merged**